docs/install/collecting-logs/README.md

```diff
@@ -0,0 +1,87 @@
+This document describes how to set up [Fluent Bit](https://docs.fluentbit.io/),
+a log processor and forwarder, to collect your kubernetes logs in a central
+directory. This is not required for running Knative, but can be helpful with
+[Knative Serving](../serving), which will automatically delete pods (and their
+associated logs) when they are no longer needed. Note that Fluent Bit support
+exporting to a number of other log providers; if you already have an existing
+log provider (for example, Splunk, Datadog, ElasticSearch, or Stackdriver), then
+you may only need the second part of setting up and configuring log forwarders.
```

Comment:
Maybe the second part can link to where you are talking about, to skip ahead?

Reply:
Done.